app/assets/javascripts/discourse/app/components/post.gjs

```diff
@@ -0,0 +1,574 @@
+import Component from "@glimmer/component";
+import { tracked } from "@glimmer/tracking";
+import { inject as controller } from "@ember/controller";
+import { concat, hash } from "@ember/helper";
+import { on } from "@ember/modifier";
+import { action } from "@ember/object";
+import { getOwner } from "@ember/owner";
+import { service } from "@ember/service";
+import { TrackedArray } from "@ember-compat/tracked-built-ins";
+import { TrackedAsyncData } from "ember-async-data";
+import { and, eq, not, or } from "truth-helpers";
+import DButton from "discourse/components/d-button";
+import ShareTopicModal from "discourse/components/modal/share-topic";
+import PluginOutlet from "discourse/components/plugin-outlet";
+import PostActionsSummary from "discourse/components/post/actions-summary";
+import PostAvatar from "discourse/components/post/avatar";
+import PostCookedHtml from "discourse/components/post/cooked-html";
+import PostEmbedded from "discourse/components/post/embedded";
+import PostLinks from "discourse/components/post/links";
+import PostMenu from "discourse/components/post/menu";
+import PostMetaData from "discourse/components/post/meta-data";
+import PostMetaDataReplyToTab from "discourse/components/post/meta-data/reply-to-tab";
+import PostNotice from "discourse/components/post/notice";
+import TopicMap from "discourse/components/topic-map";
+import concatClass from "discourse/helpers/concat-class";
+import { isTesting } from "discourse/lib/environment";
+import getURL, { getAbsoluteURL } from "discourse/lib/get-url";
+import postActionFeedback from "discourse/lib/post-action-feedback";
+import { nativeShare } from "discourse/lib/pwa-utils";
+import { applyValueTransformer } from "discourse/lib/transformer";
+import DiscourseURL from "discourse/lib/url";
+import { clipboardCopy } from "discourse/lib/utilities";
+import parentClass from "discourse/modifiers/parent-class";
+import { i18n } from "discourse-i18n";
+
+export default class Post extends Component {
+  @service appEvents;
+  @service capabilities;
+  @service currentUser;
+  @service dialog;
+  @service keyValueStore;
+  @service modal;
+  @service search;
+  @service site;
+  @service siteSettings;
+  @service store;
+
+  @controller("topic") topicController;
+
+  @tracked expandedFirstPost = false;
```

Comment:
Feels a bit weird to set this `false`, when it will end up being a Post model instance.

Could we just make it null to start with?

```suggestion
  @tracked expandedFirstPost;
```

Reply:
Done